### PR TITLE
[VEN-1282]: Finalise mainnet PSM VIP

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR finalises the mainnet [PSM VIP](https://github.com/VenusProtocol/vips/pull/28)

Resolves VEN-1282

### CODE REVIEW

File: vips/vip-131/vip-131.ts [hunk 1]
```diff
@@ -5,23 +5,31 @@ import { makeProposal } from "../../src/utils";
 
 export const FEE_OUT = 100; // 10bps
 export const FEE_IN = 0;
+export const BASE_RATE_MANTISSA = parseUnits("4", 18); // 4%
```
Comment (chechu) [inline vips/vip-131/vip-131.ts:8]:
```suggestion
export const BASE_RATE_MANTISSA = parseUnits("0.04", 18); // 4%
```

Comment (chechu) [reply vips/vip-131/vip-131.ts:8]: I would add a check in the simulation, before executing the VIP, to verify the current value. This way you will be aware of the units (in this case, the current value is 0.01e18)

File: simulations/vip-131/vip-131/simulations.ts
Comment (chechu) [inline simulations/vip-131/vip-131/simulations.ts:1]: I would verify the events emitted in the execution of the VIP

File: vips/vip-131/vip-131.ts (hunk 1 above)
Comment (chechu) [reply vips/vip-131/vip-131.ts]: Correct

File: simulations/vip-131/vip-131/simulations.ts [hunk 2]
```diff
@@ -57,7 +59,31 @@ forking(30501836, () => {
     await setMaxStalePeriodInChainlinkOracle(CHAINLINK_ORACLE, USDT, USDT_PRICE_FEED, NORMAL_TIMELOCK);
   });
 
+  describe("Pre-VIP behavior", () => {
+    it("Verify VAI base rate is 1%", async () => {
+      const currentBaseRate = await vaiControllerProxy.baseRateMantissa();
+      console.log(currentBaseRate.toString());
```
Comment (chechu) [inline simulations/vip-131/vip-131/simulations.ts:65]:
```suggestion
```

File: vips/vip-156/vip-156.ts [hunk 3]
```diff
@@ -120,11 +177,23 @@ export const vip131 = () => {
...
       {
         target: VAI_CONTROLLER_PROXY,
         signature: "setBaseRate(uint256)",
         params: [BASE_RATE_MANTISSA],
       },
+
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDT, USDT_FUNDING_AMOUNT, PSM_USDT],
```
Comment (chechu) [inline vips/vip-156/vip-156.ts:195]: I don't think we should transfer directly the USDT from the treasury to the PSM. I would transfer the USDT from the treasury to the Normal Timelock, and then I would invoke `swapStableForVAI`, setting the VTreasury address as the `receiver` address. This way we'll have VAI in the Treasury and we'll be able to recover the "deposited" USDT

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/157

**Review by kkirka — APPROVED** [review]

**Pull request merged**